recipes-wpe/wpeframework/wpeframework-ocdm-clearkey_git.bb

```diff
@@ -1,10 +1,10 @@
 SUMMARY = "WPE Framework OpenCDMi module for clearkey"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1fe8768cbb5fd322f7d50656133549de"
```

Comment:
what changed here ?

Reply:
Since the md5sum of this file is changed with the new revision of this repo

Reply:
Please add a comment to commit stating whats the reason for this checksum change.

Reply:
Will check and update